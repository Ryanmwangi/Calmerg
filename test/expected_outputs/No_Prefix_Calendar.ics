BEGIN:VCALENDAR
NAME:No Prefix Calendar
PRODID:-//CalMerge//Calendar Merger 1.0//EN
VERSION:2.0
CALSCALE:GREGORIAN
BEGIN:VEVENT
UID:20231225T000000-001@example.com
DTSTART;VALUE=DATE:20231225
DTEND;VALUE=DATE:20231226
DTSTAMP:20231225T000000Z
SUMMARY:Christmas Day
LOCATION:San Francisco
END:VEVENT
END:VCALENDAR