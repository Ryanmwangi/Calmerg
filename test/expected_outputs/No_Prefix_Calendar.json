{
  "linkGroupName": "No Prefix Calendar",
  "calendars": [
    {
      "url": "C:\\Users\\user\\OneDrive\\Desktop\\Internship_tasks\\final calmerg\\test\\test_calendars\\sf_public_holidays.ics",
      "prefix": "",
      "override": false
    }
  ]
}